server/index.js

```diff
@@ -0,0 +1,5 @@
+const app = require('./app');
+
+app.listen(app.get('port'), () => {
```

Comment:
Just for convention, .env variables are usually written in all caps, it makes it easier to understand that this variable is set as an environmental variable. Just update the name 'port' to 'PORT' 😸 That's the only change I can see for now, the rest looks good to me!